activesupport/lib/active_support/duration.rb

```diff
@@ -52,6 +52,13 @@ def ==(other)
       end
     end
 
+    # Returns the amount of seconds a duration covers as a string.
+    #
+    #   1.minute.to_s   # => "60"
+    #   1.hour.to_s     # => "3600"
+    #   1.day.to_s      # => "86400"
+    #   1.month.to_s    # => "2592000"
+    #   1.year.to_s     # => "31557600.0"
```

Comment:
Actually one example should be plenty. Otherwise you're really documenting how many seconds happen to be in a unit.

Not necessarily here, but perhaps it would be good to explain why year is a float and the others are integers?